gems/sorbet/test/snapshot/check_one_bazel.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Runs a single srb init test from gems/sorbet/test/snapshot/{partial,total}/*
+
+set -euo pipefail
+shopt -s dotglob
+
+source "gems/sorbet/test/snapshot/logging.sh"
+
+# ----- Option parsing -----
+
+test_name=$1
```

Comment:
For my own sanity, can we do some arg parsing at the top of the script?

- immediately check that we have the correct required number of args
- immediately assign positional args into variables with names
- ideally show a decent help message if something's wrong

I imagine that most of the time this won't matter because bazel will pass the right args, but it's still nice for the reader to get oriented.

Reply:
i.e., `$2` is hiding about 15 lines down

Reply:
Sure, I'll rework it to do the option parsing at the top instead.